README.md

```diff
@@ -127,7 +127,43 @@ The following optional parameters should be used in the `driver_config` for the
  - `poweron` - Power on the new virtual machine. Default: true
  - `vm_name` - Specify name of virtual machine. Default: `<suite>-<platform>-<random-hexid>`
  - `resource_pool` - Name of the resource pool to use when creating the machine. Will search first pool by default, can use value 'Resources' for none.
- - `clone_type` - Type of clone, will default to "full" to create complete copies of template. Needs a VM as template parameter, if "linked" clone desired.
+ - `clone_type` - Type of clone, will default to "full" to create complete copies of template. Other values: "linked", "instant"
+
+## Clone types
+
+### Clone type: full
+
+This takes a VM or template, copies the whole disk and then boots up the machine. Default mode of operation.
+
+### Clone mode: linked
+
+Instead of a full copy, this will use delta disks to speed up the cloning process and uses much less IO operations. The `template` parameter has to be a VM in this case, as a template is not usable. After creation of the delta disks, the machine is booted up and writes to its delta only.
```

Comment:
```suggestion
 Instead of a full copy, "linked" uses delta disks to speed up the cloning process and uses many fewer IO operations. After creation of the delta disks, the machine is booted up and writes only to its delta. Requires: `--template VM` 
```
I don't know if it is `--template VM` or `template: VM`